docs/manual/developer_guide.adoc

```diff
@@ -917,3 +917,13 @@ Check out the various `ctest` options to perform specific testing, you can rerun
 Tests are added using the add_test cmake call. Each test should finish with a 0 exit-code in case everything went well and a non-zero if something failed. Output (both stdout and stderr) are collected by ctest and stored in logs or displayed. Make sure you never hard-code a path to any tool when doing testing (or anything really) in the cmake code. Always use configuration to find all the paths and then use the respective variable.
 
 See some of the existing testing code in `cmake/SSGCommon.cmake`.
+
+=== Contribution to infrastructure code
+
+The SSG build and teplating system is mostly written in Python.
```

Comment:
teplating

Reply:
Nice catch, I have rebased to fix the typo.